common/src/main/java/com/wynntils/wc/custom/item/UnidentifiedItemStack.java

```diff
@@ -68,30 +70,49 @@ public UnidentifiedItemStack(ItemStack stack) {
         if (rarityMap == null) return;
 
         List<String> items = rarityMap.get(tier);
-        if (items == null) return;
+        if (items == null || items.isEmpty()) return;
+
+        tooltip.add(new TranslatableComponent("feature.wynntils.itemGuess.possibilities"));
+
+        Map<Integer, List<MutableComponent>> levelToItems = new TreeMap<>();
 
-        MutableComponent guesses = new TextComponent("");
         for (String item : items) {
+            ItemProfile profile = WebManager.getItemsMap().get(item);
+
+            int level = (profile != null) ? profile.getLevelRequirement() : -1;
+
             MutableComponent itemDesc = new TextComponent(item).withStyle(tier.getChatFormatting());
 
-            ItemProfile itemProfile = WebManager.getItemsMap().get(item);
-            if (ItemGuessFeature.showGuessesPrice && itemProfile != null) {
-                int level = itemProfile.getLevelRequirement();
-                int itemCost = tier.getItemIdentificationCost(level);
-                itemDesc.append(new TextComponent(" [")
-                        .append(new TextComponent(itemCost + " " + EmeraldSymbols.E_STRING)
+            levelToItems.computeIfAbsent(level, i -> new ArrayList<>()).add(itemDesc);
+        }
+
+        for (Map.Entry<Integer, List<MutableComponent>> entry : levelToItems.entrySet()) {
+            int level = entry.getKey();
+            List<MutableComponent> itemsForLevel = entry.getValue();
+
+            MutableComponent guesses = new TextComponent("  ");
+
+            guesses.append(
+                    new TranslatableComponent("feature.wynntils.itemGuess.levelline", level != -1 ? level : "?"));
```

Comment:
`levelLine` begins with spaces, but we also init `guesses` with spaces? Can we move all to levelLine, or to the component? (Perhaps the second option is better)